model/src/main/java/com/epam/esm/dto/CertificateDTO.java

```diff
@@ -0,0 +1,131 @@
+package com.epam.esm.dto;
+
+import com.epam.esm.entity.Tag;
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+public class CertificateDTO {
+
+    private int id;
```

Comment:
Why int? Potentially you can have a lot of certificates, please replace with Long